ds_judgements_public_ui/templates/includes/recent_judgments.html

```diff
@@ -2,8 +2,15 @@
   <div class="recent-judgments__container">
     <h1 id="recently-published-judgments" class="recent-judgments__header">Recently published judgments</h1>
     {% include "includes/judgment_listing.html" with judgments=recent_judgments %}
-    <p class="judgment-listing__more-info">
-      <a href="{% url 'search' %}">See all recent judgments</a>
-    </p>
+    <section class="more-info-footer" aria-labelledby="more-info-footer-heading">
```

Comment:
We don't have a `more-info-footer-heading` element for this `aria-labelledby` to make sense.

Reply:
removed from section - there is aria in button below